recipes/openconnect-sso/yum_requirements.txt

```diff
@@ -0,0 +1,36 @@
+alsa-lib
```

Comment:
Are these really necessary?

Reply:
Yes, unfortunately this package relies on both pyqt and pyqtwebengine, which bring in all these cdt dependencies. I started by adding cdt's one by one, and ended up using the same yum list as in the pyqt feedstock lol

Reply:
Yeah unfortunately that's how `yum_requirements.txt` works. Downstream feedstocks often need to relist the same dependencies